plugins/modules/ipaconfig.py

```diff
@@ -252,17 +252,11 @@
 
 
 def config_show(module):
-    _result = module.ipa_command_no_name("config_show", {})
+    _result = module.ipa_command_no_name("config_show", {"all": True})
```

Comment:
Isn't this also in PR #637?

Reply:
yes, it is in PR #637. Forgot to remove it from here.

Reply:
fixed.